msbuild/Makefile

```
@@ -373,6 +379,8 @@ all-local:: $(MSBUILD_PRODUCTS) .stamp-test-xml
 
 # make all the target assemblies build when any of the sources have changed
 $(foreach dll,$(MSBUILD_TASK_ASSEMBLIES),$(dll)/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).dll $(dll)/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).pdb): .build-stamp
+	echo $^
+	ls -la $^
```

Still not quite right :smile:

You want this (and only this) line: https://github.com/rolfbjarne/xamarin-macios/commit/eb167b344647f60c21ada79120a8cc701473850a#diff-14d3973c482862a4cc5f2663d281bbb6R382

```suggestion
	$(foreach dll,$(TRANSLATED_ASSEMBLIES),Xamarin.Localization.MSBuild/bin/$(CONFIG)/$(TARGETFRAMEWORK)/$(dll).resources.dll): .build-stamp
```

---

Okay I committed your suggestion!